index.html

```diff
@@ -956,6 +956,18 @@ <h1 class="text-white">About "In the Heat of the Code"</h1>
 						<div class="col-md-3 col-sm-6 speaker-column">
 							<div class="speaker">
 								<div class="image-holder">
+									<img class="background-image" alt="Tarun Kumar" src="img/TarunKumar.jpg">
```

Comment:
there are details of two mentors in one column please check it agian

Reply:
I can't find that. Everything working fine locally!

Reply:
@isuruAb Can you provide a screenshot or a correction?

Reply:
@niccokunzmann  send a pr to @meets2tarun  fork. 👍